.github/workflows/publish.yml

```diff
@@ -51,6 +51,10 @@ jobs:
           # Remove JSON files from source destination
           find examples -type f -name "*.json" -delete
 
+          # Copy across new version to maintain a stable reference URL at /latest
+          rm -rf latest/*
```

Comment:
Removal of old files is necessary as a new version won't necessarily mean just existing files changing and new ones being added - we also want to clear out files which have been deleted as part of a new release.